db/migrate/20180920225744_add_archived_to_distribution_points.rb

```diff
@@ -0,0 +1,5 @@
+class AddArchivedToDistributionPoints < ActiveRecord::Migration[5.1]
+  def change
+    add_column :distribution_points, :archived, :boolean, default: false
+  end
+end
```

Comment:
This is unnecessary. The `archived` column is already part of the distribution_points table. See [structure.sql#209](https://github.com/hurricane-response/florence-api/blob/master/db/structure.sql#L209) on master.

Reply:
Huh, nvm - I take this one back. Seems it's in structure.sql, but [was never in the migration](https://github.com/hurricane-response/florence-api/blob/master/db/migrate/20180919043949_create_distribution_points.rb#L18), which is odd. Not sure how that happened.